practice-areas/engineering/drupal/onboarding-materials/drupal-for-drupal-engineers.md

```diff
@@ -0,0 +1,115 @@
+
+Drupal Skills Categories: https://app.skills-base.com/skillcategories/view/id/16
+Skills Assessment: https://app.skills-base.com/people/view#assessments
+
+### Gotta Drupal problem? We can help!
+
+One of the most cherished practices is the "30 minute rule" which states, "You are only allowed to be stuck on a problem for 30 minutes before you ask for help."
+
+The #engineering-drupal channel in Slack is used to reach out to other CivicActions Drupal engineers to ask for help on specific Drupal problems.
+
+When you ask for help, it's a good idea to provide a context for your problem, describe (or display) the problem, then describe what approaches you've already taken to solve the problem. If you can summarize the nature of the problem in the first sentence, it's more likely you'll get a quick response.
+
+CivicActions engineers tend to be very helpful, so it's rare for a question not to get a response within a few hours. If for some reason you don't get a response, try a different wording or expand on things you have tried.
+
+### Contributions to drupal.org
+
+We encourage all CivicActioners to give back to the Drupal community (see community participation). To track those contributions, follow the instructions below. Doing this helps our Practice Area to be aware of and appreciate your contributions. It also helps our organization as a whole!
+
+Update your drupal.org profile
+
+1. Go to http://drupal.org/.
+2. Log into your drupal.org account (create one if you do not have one).
+3. Align it with CivicActions in the "Work" section: Edit profile > Then expand the Work section > Type in "CivicActions" and your Job title.
+4. Align your role with CivicActions in the "Contributor roles" section: Edit profile > Then expand the Contributor roles section > Type in "CivicActions" in "Organization support". See https://www.drupal.org/community/contributor-guide/find-your-role for more details.
+
+### Contribution to drupal.org modules and themes
+
+If you have created or are creating a module or a theme and use CivicActions time to work on it — either for a client or as professional development/community participation time — please make sure to add CivicActions under 'Supporting organizations' when editing the module or theme. An example screenshot of what the checkbox looks like:
+
+![supporting organizations](/assets/images/drupal-pa/do-supporting-organizations.png)
+
+Adding the organization will ensure that the module and theme will also be attributed to CivicActions. Examples of that can be seen under https://www.drupal.org/civicactions#projects-supported.
+
+### Contributing to drupal.org issues
+
+If you are contributing to drupal.org issues for CivicActions and our customers, please make sure to click the box under 'Attribute this contribution'. An example screenshot of what the checkbox looks like:
+
+![attribute organization](/assets/images/drupal-pa/do-attribute-contribution.png)
```

Comment:
The image links needs to be setup to navigate to the folder. For an example, checkout the source of https://guidebook.civicactions.com/en/latest/practice-areas/engineering/drupal/.